src/connector/src/sink/doris.rs

```diff
@@ -125,8 +127,11 @@ impl DorisSink {
             .collect();
 
         let rw_fields_name = self.schema.fields();
-        if rw_fields_name.len().ne(&doris_columns_desc.len()) {
-            return Err(SinkError::Doris("The length of the RisingWave column must be equal to the length of the doris column".to_string()));
+        if rw_fields_name.len().gt(&doris_columns_desc.len()) {
```

Comment:
Use `>` here instead of `gt`. 

And should we allow `gt` only when partial update is enabled in the with options?

Reply:
In theory, these are two things, gt thought that the column of rw was a downstream subset, while partial_update means opening starrocks/doris's feature